config/jobs/kubernetes/sig-scalability/sig-scalability-golang.yaml

```diff
@@ -91,8 +91,8 @@ periodics:
       args:
       - --cluster=gce-golang
       - --env=CL2_LOAD_TEST_THROUGHPUT=50
-      - --env=KUBEMARK_CONTROLLER_MANAGER_TEST_ARGS=--profiling --kube-api-qps=200 --kube-api-burst=200
-      - --env=KUBEMARK_SCHEDULER_TEST_ARGS=--profiling --kube-api-qps=200 --kube-api-burst=200
+      - --env=KUBEMARK_CONTROLLER_MANAGER_TEST_ARGS=--profiling --contention-profiling --kube-api-qps=200 --kube-api-burst=200
+      - --env=KUBEMARK_SCHEDULER_TEST_ARGS=--profiling --contention-profiling --kube-api-qps=200 --kube-api-burst=200
```

Comment:
It is worth noting that both `--profiling` and `--contention-profiling` flags are deprecated for `kube-scheduler`, see https://kubernetes.io/docs/reference/command-line-tools-reference/kube-scheduler/

Do we still want to keep them here?